src/mui/view/TabbedView.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import compose from 'recompose/compose';
+import { Tabs, Tab } from 'material-ui/Tabs';
+import getDefaultValues from '../form/getDefaultValues';
+import translate from '../../i18n/translate';
+
+export class TabbedView extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            value: 0,
+        };
+    }
+
+    handleChange = (value) => {
+        this.setState({ value });
+    };
+
+    render() {
+        const { children, contentContainerStyle, record, resource, basePath, translate } = this.props;
+        return (
+            <div style={{ padding: '0 1em 1em 1em' }}>
```

Comment:
no object literal in JSX. Please read https://marmelab.com/blog/2017/02/06/react-is-slow-react-is-fast.html#beware-of-object-literals-in-jsx